### DISCUSSION TITLE
build error of eide

### DISCUSSION THREAD

**Discussion by coochew** [github0null/eide#322] [General]
Error: A9555E: Failed to check out a license.
how to fix it ?

**Comment by github0null** [comment]
You need a armcc compiler license before use it.

Open your keil license management program, and register your license,

and then retry.

**Comment by coochew** [comment]
but I not in using keil, I'am using the vscode plug in  --- embedded IDE-------- 原始邮件 --------发件人： null ***@***.***>日期： 2024年4月1日周一 09:57收件人： github0null/eide ***@***.***>抄送： coochew ***@***.***>, Author ***@***.***>主    题： Re: [github0null/eide] build error of eide (Discussion #322)
You need a armcc compiler license before use it.
Open your keil license management program, and register your license,
and then retry.
